### DISCUSSION TITLE
SCSS to CSS

### DISCUSSION THREAD

**Discussion by Abhisheknanda1344463** [GrapesJS/grapesjs#5116] [Q&A]
Hi @artf 
Is there any way to support SCSS in grapesJs 
if I set the editable property of the code mirror and wrote SCSS syntax in the CSS part it will reflect the editor
with this command 
`editor.setStyle(scss || css`)

I am using Vue js with type
I've tried with **sass** package it gives me the error **node: { fs: "empty" **

**Comment by artf** [comment]
I guess the simple solution would be to compile SCSS to CSS on your side and then pass it to GrapesJS
